samples/src/main/java/com/example/bigquery/ListModels.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+import com.google.api.gax.paging.Page;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQuery.ModelListOption;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Model;
+
+// [START bigquery_list_models]
+public class ListModels {
+
+  public static void runListModels() {
+    // TODO(developer): Replace these variables before running the sample.
+    String datasetName = "MY_DATASET_NAME";
+    listModels(datasetName);
+  }
+
+  public static void listModels(String datasetName) {
+    try {
+      // Initialize client that will be used to send requests. This client only needs to be created
+      // once, and can be reused for multiple requests.
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+
+      Page<Model> models = bigquery.listModels(datasetName, ModelListOption.pageSize(100));
+      if (models == null) {
+        System.out.println("Dataset does not contain any models");
```

Comment:
nit: You should use if/break over if/else here. It simplifies the control flow and makes it easier to read, and easier for people to write code after the if/else block that might rely on `models` (since it's guaranteed to be non-null at that point). 

```java
if (models == null) {
  System.out.println("Dataset does not contain any models.");
  return;
}
System.out.printf("Dataset contains %s models:\n", models.getLength());
models
  .iterateAll()
  .forEach(
     model -> System.out.printf("Model ID: %s", model.getModelId()));
```

Reply:
okay - updated